src/components/Header/Header.tsx

```diff
@@ -1,13 +1,25 @@
-import ConnectButton from '../Button/ConnectButton'
 import useStyles from './Header.styles'
 
 function Header() {
 	const classes = useStyles()
 
 	return (
 		<div aria-hidden className={classes.header}>
-			<span>Peeps Club</span>
-			<ConnectButton />
+			<span className={classes.title}>Peeps Club</span>
+			<nav>
+				<a href="#" className={classes.link}>
+					Passport
+				</a>
+				<a href="#" className={classes.link}>
+					Peep
+				</a>
+				<a href="#" className={classes.link}>
+					Academy
+				</a>
+				<a href="#" className={classes.link}>
+					Coming Soon
+				</a>
```

Comment:
These links won't be ready before launch. Can we update to Twitter, Discord, Etherscan?

Reply:
https://twitter.com/Peeps_Club
https://discord.gg/peepsclub
TBA after deployment